contrib/bdii/htcondor-ce-provider

```diff
@@ -155,7 +156,7 @@ def main():
     """
 
     # Get hostname of the batch system central manager
-    central_manager = htcondor.param.get('COLLECTOR_HOST').split(' ')[0]
+    central_manager = re.split(r'\s*,\s*', htcondor.param.get('COLLECTOR_HOST'))[0]
```

Comment:
```suggestion
    central_manager = re.split(r'\s*[\s,]\s*', htcondor.param.get('COLLECTOR_HOST'))[0]
```

The PR title says "_Add_ support for comma-separated central managers"

And, correspondingly, your commit message says "Split on whitespace _or_ commas"

but as hard as i squint at this regex the comma does not look optional.

Is it your intention to remove support for whitespace-only -separated names?

....

Alternatively if you want to allow collapsing multiple commas as a single separator, you can do

```
    central_manager = re.split(r'[\s,]+', htcondor.param.get('COLLECTOR_HOST'))[0]
```

Reply:
Carl's second suggested alternative matches what Condor does internally.